meson.build

```diff
@@ -422,7 +422,7 @@ need_glx_functions = false
 if ['auto', 'gl'].contains(gl_backend)
   gl_dep = dependency('gl', required: gl_backend == 'gl')
   if not gl_dep.found()
-    if cc.has_header('GL/gl.h')
+    if cc.has_header('GL/gl.h') and host_machine.system() != 'windows'
```

Comment:
Actually, not sure why this is needed. `gl_dep.found()` is always true when building for Windows because of [a Meson bug](https://github.com/mesonbuild/meson/issues/3679).

Reply:
You mean it won't be necessary **after** using `method: 'pkg-config'`?

Reply:
> You mean it won't be needed **after** using `method: 'pkg-config'`?

 Ah, I looked at the actual bug (https://github.com/mesonbuild/meson/pull/3708), maybe there is difference between native builds and crossbuilds in this case (https://github.com/ystreet/meson/commit/158bd7d2e7897393e017ed7d8a97877544125fd6#diff-be4e5967e4fb2136a4f9d2a4da16e351L49).